src/packages/core/components/input-with-alias/input-with-alias.element.ts

```diff
@@ -68,17 +68,21 @@ export class UmbInputWithAliasElement extends UmbFormControlMixin<string>(UmbLit
 	}
 
 	render() {
-		// Localizations: [NL]
 		return html`
-			<uui-input id="name" placeholder="Enter a name..." label=${this.label} .value=${this.value} @input="${this.#onNameChange}">
+			<uui-input
+				id="name"
+				placeholder=${this.localize.term('placeholders_entername')}
+				label=${this.label}
+				.value=${this.value}
+				@input=${this.#onNameChange}>
 				<!-- TODO: should use UUI-LOCK-INPUT, but that does not fire an event when its locked/unlocked -->
 				<uui-input
 					name="alias"
 					slot="append"
 					label="alias"
```

Comment:
Looks good, I think we should copy the placeholder localization to this label attribute as well :)